libsrc/osic1p/clrscr.s

```diff
@@ -0,0 +1,31 @@
+;
+; void clrscr (void);
+;
+        .export         _clrscr
+        .import         plot
+        .include        "extzp.inc"
+        .include        "osic1p.inc"
+
+; Adapted from the Challenger Character Graphics
+; Reference Manual, "2.3.3 MACHINE LANGUAGE SCREEN CLEAR"
+; This is self-modifying code!
+BANKS = VIDEORAMSIZE / $100
+
+_clrscr:
+        lda       #$20          ; ' '

```

Comment:
> please be aware that the assembler allows to directly define ASCII constants...

Understood. I'm getting slowly more proficient in 6502 assembler...

Thanks for merging the pull request!